### PR TITLE
Docs: Add windows dev setup doc

### PR DESCRIPTION
**Pull request by daltonrpruitt**
Based on our experiences with getting my stuff working...

Co-authored-by: Zee <50284+zspencer@users.noreply.github.com>
Co-authored-by: Ana <anaulin@users.noreply.github.com>
Co-authored-by: Dalton <daltonrpruitt@user.noreply.github.com>
Co-authored-by: Kelly Hong <KellyAH@user.noreply.github.com>

### CODE REVIEW

**Comment by daltonrpruitt** [comment]
I assign all rights, including copyright, to any of my future work on Zinc projects to Zinc Collective LLC

**Review by anaulin — APPROVED** [review]
Thanks for documenting this!

Totally optional: would be nice to also add a link to this doc from https://github.com/zinc-collective/convene/blob/main/CONTRIBUTING.md#21-developer-setup-and-documentation, so that other folks know to find this.

Feel free to merge it when you feel ready!

**Comment by daltonrpruitt** [comment]
The link from the CONTRIBUTING doc was actually something @zspencer mentioned we should have that I completely forgot to do yesterday. Thank you for the reminder!

**Comment by daltonrpruitt** [comment]
It currently seems it needs to be merged by someone with write access. Do I need to merge to my fork first?

**Review by zspencer — APPROVED** [review]
Love it! Thank you!

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -75,7 +75,7 @@ Designers who have agreed to protect the privacy of our clients may access our
 ### 2.1. Developer Setup and Documentation.
 
 To set up your development environment you'll need `ruby`, `node` and
-`postgresql`.
+`postgresql`. If on Windows using WSL2, please see [this document](docs/windows-wsl-dev-setup.md).
```
Comment (zspencer) [inline CONTRIBUTING.md:78]: Part of me thinks this should be a 

```markdown

> _For Windows users, see our [Windows Developer Setup](docs/windows-wsl-dev-setup.md)]

```

to set it apart from the main body; but I think it's fine as is and am merging !

**Pull request merged**